include/TrustWalletCore/TWElrondNetworkConfig.h

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2017-2020 Trust Wallet.
```

Comment:
Not sure if Trust owns the copyright of this file, even if it was added to a fork of their repository.

Reply:
The header is a requirement of the contributions, as well.